api/session_keys/module.go

```diff
@@ -51,7 +51,10 @@ func (m Module) GenerateSessionKeys(dataPtr int32, dataLen int32) int64 {
 	b := m.memUtils.GetWasmMemorySlice(dataPtr, dataLen)
 	buffer := bytes.NewBuffer(b)
 
-	seed := sc.DecodeOptionWith(buffer, sc.DecodeSequence[sc.U8])
+	seed, err := sc.DecodeOptionWith(buffer, sc.DecodeSequence[sc.U8])
+	if err != nil {
+		return m.memUtils.BytesToOffsetAndSize([]byte(err.Error()))
```

Comment:
I think the Host does not expect an error to be returned.
Call `log.Critical(err.Error())` instead

Reply:
Sorry, seems I've forgotten to remove it. Done now.